frontend/packages/ceph-storage-plugin/src/components/modals/clone-pvc-modal/_clone-pvc-modal.scss

```diff
@@ -0,0 +1,12 @@
+.ceph-clone-pvc-modal__details-section {
+  margin-top: var(--pf-global--spacer--xl);
+  .ceph-clone-pvc-modal__details {
+      display: flex;
+      flex-direction: row;
+      justify-content: space-between;
+  }
+
+  .ceph-clone-pvc-modal__pvc-details {
+    font-weight: 600;
```

Comment:
Why do we need to adjust the `font-weight`?

Reply:
these texts(pvc details title) needs to be bold as per UXD. Hence, I have used this.